tests/test_helpers.py

```diff
@@ -53,6 +53,9 @@
     (["helper", "l_projecta", "--all"],
      "20ly_newprojectum\n20sb_firstprojectum\n"
```

Comment:
it looks as if we are not properly testing this.  We need a prum in the schema exemplar that is not current to properly test the functionality.  I guess ideally we would have prums in there that have each status to make sure the filters work properly.  This will probably cause all kinds of tests to fail on other helpers, but I think it is worth putting in the effort now to get the tests good.  rn they are not really testing much of the desired behaviors.

Actually, the first step may be for us to really decide what we want the behavior to be, make UCs then code up the tests, then finally work on the code....proper TDD